app/src/main/java/com/assist_software/expenseappmvp/screens/addActionScreen/AddActionView.kt

```diff
@@ -25,33 +30,50 @@ import io.reactivex.subjects.PublishSubject
 import kotlinx.android.synthetic.main.activity_add_action.view.*
 import kotlinx.android.synthetic.main.transaction_toolbar.view.*
 import java.io.ByteArrayOutputStream
+import java.text.SimpleDateFormat
 import java.util.*
 
-class AddActionView(var activity: AddActionActivity) {
+class AddActionView(var activity: AddActionActivity) : DatePickerDialog.OnDateSetListener,
+    TimePickerDialog.OnTimeSetListener {
     val layout: View = View.inflate(activity, R.layout.activity_add_action, null)
-    private val c = Calendar.getInstance()
+    private var day = 0
```

Comment:
Add an class that contains all of this info, and create a instance instead of using all of this local variables in view